src/main/kotlin/uk/gov/justice/digital/hmpps/externalusersapi/config/HmppsExternalUsersApiExceptionHandler.kt

```diff
@@ -251,6 +253,34 @@ class HmppsExternalUsersApiExceptionHandler {
         )
       )
   }
+
+  @ExceptionHandler(UserGroupException::class)
+  fun handleAuthUserGroupException(e: UserGroupException): ResponseEntity<ErrorResponse> {
+    log.debug("Auth user group exception caught: {}", e.message)
```

Comment:
handler and logging should not reference Auth user (same on line 272/273)

Reply:
Removed Auth references